osu.Game/Users/Country.cs

```diff
@@ -1,27 +1,768 @@
 ﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
-#nullable disable
-
-using System;
+using System.ComponentModel;
+using JetBrains.Annotations;
 using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
 
 namespace osu.Game.Users
 {
-    public class Country : IEquatable<Country>
+    [JsonConverter(typeof(StringEnumConverter))]
+    [UsedImplicitly(ImplicitUseTargetFlags.WithMembers)]
+    public enum Country
     {
-        /// <summary>
-        /// The name of this country.
-        /// </summary>
-        [JsonProperty(@"name")]
-        public string FullName;
-
-        /// <summary>
-        /// Two-letter flag acronym (ISO 3166 standard)
-        /// </summary>
-        [JsonProperty(@"code")]
-        public string FlagName;
-
-        public bool Equals(Country other) => FlagName == other?.FlagName;
+        [Description("Alien")]
+        XX = 0,
```

Comment:
I initially went towards using `null` as a default/unknown country value, but wasn't sure if that's a good direction code-wise.

I was also not sure between using `Country.XX` and `default`, but decided to use the latter as it reads somewhat better. Open for argument.